editor/editor_properties.cpp

```diff
@@ -790,262 +790,293 @@ EditorPropertyFlags::EditorPropertyFlags() {
 }
 
 ///////////////////// LAYERS /////////////////////////
+void EditorPropertyLayersGrid::_rename_pressed(int p_menu) {
+	// Show rename popup for active layer.
+	if (renamed_layer_index == -1) {
+		return;
+	}
+	String name = names[renamed_layer_index];
+	rename_dialog->set_title(TTR("Renaming layer " + itos(renamed_layer_index + 1) + ":"));
```

Comment:
Partial strings can't be translated properly, you need to use string formatting with placeholders for this:
```suggestion
	rename_dialog->set_title(vformat(TTR("Renaming layer %d:"), renamed_layer_index + 1));
```